packages/web/src/pages/audio-rewards-page/components/SendInputBody.tsx

```diff
@@ -298,13 +298,9 @@ const SendInputBody = ({
           onChange={handleChangeAddress}
         />
         {renderAddressError()}
-        <Button
-          className={styles.sendBtn}
-          text={messages.sendAudio}
-          textClassName={styles.sendBtnText}
-          onClick={onClickSend}
-          type={hasError ? ButtonType.DISABLED : ButtonType.PRIMARY_ALT}
-        />
+        <Button variant='primary' onClick={onClickSend} disabled={!!hasError}>
```

Comment:
```suggestion
        <Button variant='primary' onClick={onClickSend} disabled={hasError}>
```

Reply:
sadly hasError is a string enum, so this is to cast it to boolean type